src/empty-placeholder.ts

```diff
@@ -113,6 +120,16 @@ export class EmptyPlaceholder extends LitElement {
     `;
   }
 
+  private get collectionErrorTemplate(): TemplateResult {
```

Comment:
q: does this fire a sentry error or anything when displayed? should we?

Reply:
We fire a Sentry message whenever we get an error back from the PPS, so this is already covered.